ui/main/src/app/business/services/user.service.ts

```diff
@@ -166,10 +195,28 @@ export class UserService extends CrudService {
     }
 
     loadConnectedUsers(): Observable<any[]> {
-        return this.httpClient.get<any[]>(`${this.connectionsUrl}`);
+        return this.userServer.loadConnectedUsers().pipe(
+            map((userResponse) => {
+                if (userResponse.status === ServerResponseStatus.OK){
+                    return userResponse.data;
+                } else {
+                    this.handleServerResponseError(userResponse);
+                    return [];
+                }
+            })
+            );
     }
 
     willNewSubscriptionDisconnectAnExistingSubscription(): Observable<boolean> {
-        return this.httpClient.get<boolean>(`${this.willNewSubscriptionDisconnectAnExistingSubscriptionUrl}`);
+        return this.userServer.willNewSubscriptionDisconnectAnExistingSubscription().pipe(
+            map((userResponse) => {
+                if (userResponse.status === ServerResponseStatus.OK){
+                    return userResponse.data;
+                } else {
+                    this.handleServerResponseError(userResponse);
```

Comment:
In this case , do not use handleServerResponse but the logger : 
logger.error("Impossible to check if new connection will disconnect existing subscription ")